apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -59,11 +72,19 @@ public SpannerCommand()
         }
 
         /// <summary>
+        /// Initializes a new instance of <see cref="SpannerCommand"/>
         /// </summary>
-        /// <param name="commandTextBuilder"></param>
-        /// <param name="connection"></param>
-        /// <param name="transaction"></param>
-        /// <param name="parameters"></param>
+        /// <param name="commandTextBuilder">The <see cref="SpannerCommandTextBuilder"/>
+        /// that configures the text of this command.</param>
+        /// <param name="connection">The <see cref="SpannerConnection"/> that is
+        /// associated with this <see cref="SpannerCommand"/></param>
+        /// <param name="transaction">An optional <see cref="SpannerTransaction"/>
+        /// created through <see>
+        /// <cref>SpannerConnection.BeginTransactionAsync</cref>
+        /// </see>
```

Comment:
I believe its the same thing (and you dont need the redundant text), but will give it a test with doxygen which is how I created my last set of C# docs.